lib/view/stream_device_view.dart

```diff
@@ -201,6 +201,7 @@ class _StreamDeviceViewState extends State<StreamDeviceView> {
     log.info('device selected: $device');
     if (device != null) {
```

Comment:
Please check if device always null. as we already handle canvas deeplink in deeplink service.